jobs/build-snaps/build-release-cdk-addons.groovy

```diff
@@ -7,7 +7,7 @@ def kube_ersion = null
 if (kube_version != "") {
     kube_ersion = kube_version.substring(1)
 }
-def lxc_name = "build-release-cdk-addons-"+env.BUILD_NUMBER
+def lxc_name = env.JOB_NAME.replaceAll('\\.', '-')+"-"+env.BUILD_NUMBER
```

Comment:
groovy `replaceAll` takes a regex as the first arg, so we need a backslash to escape the backslash that escapes the dot.

Reply:
gotcha. container names don't like `.`.  So this changes `.` to `-` using regex.